services/web/server/tests/unit/test_activity.py

```diff
@@ -0,0 +1,118 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+import importlib
+import json
```

Comment:
MINOR: cleanup unused imports  otherwise pylint check will fail. Can see this directly in your editor with the right setup:

![image](https://user-images.githubusercontent.com/32402063/69241717-98d4a600-0b9f-11ea-85f4-396c79d612cc.png)

MINOR: sort imports

Reply:
can you help me with that setup please? I had it working before, I don't know what happened...